walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,65 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredOptional1() {
```

Comment:
You should have descriptive names instead of fooBar1, fooBar2, etc... The reader should know what the test is doing by the name. Which is hard for me to tell right now. Generally you have the base test name followed by an underscore and then some state or expected output (e.g testMethodName_InvalidArguments, testMethodName_ValidArguments, testMethodName_SomeWeirdEdgeCase...)

Google naming conventions or see what you can come up with yourself.

nit: Not sure what the style guide says offhand, but we usually prefix every method name with "test" (even if it is redundant)